hugo/themes/dora-team/assets/core/README.md

```diff
@@ -0,0 +1 @@
+Template core assets that are required for the DORA theme
```

Comment:
We can delete this whole folder... Core is only a thing on dora.dev

Reply:
Done